content/en/docs/contribute/style/style-guide.md

```diff
@@ -29,7 +29,8 @@ and representing feature state.
 
 ## Language
 
-Kubernetes documentation uses US English.
+Kubernetes documentations are now available in multiple languages, see [here](https://github.com/kubernetes/website/blob/master/README.md#localization-readmemds).
```

Comment:
I believe this statement refers to the use of the U.S. English spelling and grammar in the English-language version of the documentation -- in contrast to U.K English spelling and grammar. I agree that it would be useful to mention that the documentation has been translated to multiple languages, but the purpose of the above statement was actually not to recognize English as an authoritative source, but rather to instruct English writers of the appropriate spelling and grammar to use when contributing.

Reply:
Consider modifying the above to say the following:

```
Kubernetes documentation has been translated into multiple languages
(see [here](https://github.com/kubernetes/website/blob/master/README.md#localization-readmemds).

The English-language documentation uses U.S. English spelling and grammar.
```

Reply:
Yes,I missed the real point. Thanks for correcting that!